code-review/README.md

```diff
@@ -15,27 +15,47 @@ guidelines to follow during the review process.
+* Keep it real and fun. Use emojies, gifs, etc. :+1:
+* Give back your time to code review, like others do for you.
 
 ### Having Your Code Reviewed
 
-- Use the Pull Request template in your favor.
-- Add the corresponding JIRA ticket to each Pull Request.
-- Add detailed reproduction/user-flow steps that are easy to follow.
-- Add "potentially impacted"/"vulnerable areas" in additional to reproduction
+* Use the Pull Request template in your favor.
+* Add the corresponding JIRA ticket to each Pull Request.
+* Add detailed reproduction/user-flow steps that are easy to follow.
+* Add "potentially impacted"/"vulnerable areas" in additional to reproduction
   steps.
-- Be grateful and appreciative to the code reviewer.
-- Help to clear any misconceptions the reviewer may have.
-- Wait until your code has the `Status: Review Complete` label.
-- Once your code has the Review Complete label, you can add the 'Status: Ready to Ship' label to signify it's ready for release, assuming you've clarified it's ready to ship with stakeholders (if any). 
+* Be grateful and appreciative to the code reviewer.
+* Help to clear any misconceptions the reviewer may have.
+* Wait until your code has the `Status: Review Complete` label.
+* Once your code has the Review Complete label, you can add the 'Status: Ready to Ship' label to signify it's ready for release, assuming you've clarified it's ready to ship with stakeholders (if any).
 
 ### Reviewing Code
 
-- Understand the change. Understand _why_ the change is being made.
-- Identify areas for refactoring that still solve the problem.
-- Determine if the author needs to add descriptive comments to their classes. If the code is self-documenting it may be skipped. 
-- Add the appropriate label after you have reviewed and approved
-  (`Status: Reviewed`, `Status: Review Complete`).
+**What are we trying to accomplish while reviewing code?**
+
+1.  Ensure the code functions as intended.
+2.  Catch any bugs the code causes.
+3.  Ensure the code is readable.
+4.  Ensure the code is maintainable.
+5.  Ensure the code is secure.
+6.  Spread your knowledge throughout the team.
+
+**What is our code checklist**
+
+* [ ] Does the code function?
+* [ ] Is the code understandable?
+* [ ] Is the code maintainable? Modular?
+* [ ] Does the code conform to ProctorU standards and conventions.
+* [ ] Does the code have redundancy? Is there related code that can be deleted?
+* [ ] Does the code have performance issues?
+* [ ] Does the code have security vulnerabilities?
+* [ ] Can I remove any logging from the code?
+* [ ] Does TomDoc documentation exist where it is needed?
+* [ ] Is the code testable?
+* [ ] Is the code tested? System/Unit
+* [ ] Are there more features than necessary? Does this code do more than simply solving the problem? ([KISS Principle](https://en.wikipedia.org/wiki/KISS_principle))
+* [ ] Add the appropriate label after you have reviewed and approved
```

Comment:
prefer to remove this last bullet point because 1) it is not written as a question like the rest of the list, and 2) it suddenly shifts focus to the reviewer whereas the rest of the list focuses on the content of the code. this line would fit in better with the "everyone" section.